torch/onnx/symbolic.py

```diff
@@ -631,57 +712,55 @@ def abs(g, self):
 
 
 def pow(g, self, exponent):
+    exponent = _maybe_get_scalar(exponent)
     return g.op("Pow", self, _if_scalar_type_as(exponent, self), **_broadcast_if_scalar(exponent))
 
 
+@parse_args('v', 'f', 'f')
 def clamp(g, self, min, max):
     return g.op("Clip", self, min_f=min, max_f=max)
 
 
+@parse_args('v', 'f')
 def clamp_min(g, self, min):
     return g.op("Clip", self, min_f=min)
 
 
+@parse_args('v', 'f')
 def clamp_max(g, self, max):
     return g.op("Clip", self, max_f=max)
 
 
 # torch.max (same for torch.min) actually has two interfaces smashed together:
 # torch.max(x, dim, keepdim) and torch.max(x, y)
-def max(g, self, *args, **kwargs):
-    dim = kwargs.get("dim", None)
-    if dim is None and isinstance(args[0], numbers.Number):
-        dim = args[0]
-    if dim is not None:
-        keepdim = kwargs.get("keepdim", False)
+def max(g, self, dim_or_y, keepdim=None):
+    if keepdim is None:
+        return g.op("Max", self, dim_or_y)
+    else:
+        dim = _get_const(dim_or_y, 'i', 'dim')
+        keepdim = _get_const(keepdim, 'i', 'keepdim')
         # TODO: export it as ReduceMax
         return g.op("ATen",
                     self,
                     operator_s="max",
                     dim_i=dim,
                     keepdim_i=keepdim,
                     outputs=2)
-    else:
-        (other,) = args
-        return g.op("Max", self, other)
 
 
-def min(g, self, *args, **kwargs):
-    dim = kwargs.get("dim", None)
-    if dim is None and isinstance(args[0], numbers.Number):
-        dim = args[0]
-    if dim is not None:
-        keepdim = kwargs.get("keepdim", False)
-        # TODO: export it as ReduceMin
+def min(g, self, dim_or_y, keepdim=None):
```

Comment:
Because we want to work with raw values. `parse_args` is there only to automatically convert Values into constants (so that we don't have to change most of the symbolics, which previously assumed those would be delivered as constants via attributes)